google/auth/external_account.py

```diff
@@ -439,6 +439,7 @@ def validate_token_url(token_url, url_type="token"):
         _TOKEN_URL_PATTERNS = [
             "^[^\\.\\s\\/\\\\]+\\.sts\\.googleapis\\.com$",
             "^sts\\.googleapis\\.com$",
+            "^sts\\.mtls\\.googleapis\\.com$",
```

Comment:
Do we need to support regional mtls endpoints?

Reply:
Let me check if that is the case

Reply:
I can confirm regional MTLS endpoints should also be supported. Will follow up.